ScoutSuite/providers/azure/rules/findings/storageaccount-default-network-access.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
What's the difference between this one and `ScoutSuite/providers/azure/rules/findings/storageaccount-public-blob-container.json`?

Reply:
If I am not wrong, the ScoutSuite/providers/azure/rules/findings/storageaccount-public-blob-container.json refers to 3.6 point (Ensure that 'Public access level' is set to Private for blob containers) and this refers to 3.7 (Ensure default network access rule for Storage Accounts is set to deny)